Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -153,28 +172,109 @@ def create_pulse_by_ec(entry: dict) -> dict:
     return assign_params(**pulse_by_ec)
 
 
-def create_attack_pattern_relationships(client: Client, raw_response: dict, entity_a: str, entity_a_type: str):
+def extract_attack_ids(raw_response: dict):
+    """
+    extract the attack_ids field from the raw response if exists
+
+    Args:
+        raw_response: The response gotten from the server
+
+    :returns:
+        the attack_ids field if exists, otherwise, an empty string.
+    """
+    attack_id_field = dict_safe_get(raw_response, ['pulse_info', 'pulses'], [{}])
+    return dict_safe_get(([{}] if attack_id_field == [] else attack_id_field)[0], ['attack_ids'], [''])
```

Comment:
```suggestion
    attack_id_field = dict_safe_get(raw_response, ['pulse_info', 'pulses']) or [{}]
    return attack_id_field[0].get('attack_ids', [])
```

Reply:
See my original change suggestion, If you use the `or` in the first line you don't need the if in the second.